### PR TITLE
Set expiry when invitation deleted

### PR DESCRIPTION
**Pull request by apoorv1316**
What
- Show the error message when the invitation is deleted
Screenshot 2024-01-03 at 7 40 20 AM

### CODE REVIEW

File: app/controllers/internal_api/v1/invitations_controller.rb [hunk 1]
```diff
@@ -19,6 +19,7 @@ def update
   def destroy
     authorize @invitation
 
+    @invitation.mark_as_deleted
```
Comment (prasanthchaduvula) [inline app/controllers/internal_api/v1/invitations_controller.rb:22]: Remove this `mark_as_deleted`. Not required

File: app/models/invitation.rb [hunk 2]
```diff
@@ -90,6 +90,11 @@ def set_token
     end
   end
 
+  def mark_as_deleted
```
Comment (prasanthchaduvula) [inline app/models/invitation.rb:93]: Remove this not required

**Review by prasanthchaduvula — CHANGES_REQUESTED** [review]

File: app/controllers/internal_api/v1/invitations_controller.rb (hunk 1 above)
Comment (apoorv1316) [reply app/controllers/internal_api/v1/invitations_controller.rb]: done

File: app/models/invitation.rb (hunk 2 above)
Comment (apoorv1316) [reply app/models/invitation.rb]: done

**Review by prasanthchaduvula — APPROVED** [review]

**Pull request merged**